### PR TITLE
#729: Check on new size quantity

### PR DESCRIPTION
**Pull request by hani-ibrahim**
fixing #729

### CODE REVIEW

**Comment by hani-ibrahim** [comment]
👍

**Comment by hani-ibrahim** [comment]
👍

File: AtlasUI/AtlasUI/CheckoutSummaryEditProductStackView.swift [hunk 1]
```diff
@@ -67,11 +67,13 @@ class CheckoutSummaryEditProductStackView: UIStackView {
 
     private func sizeSelected(at idx: Int, for selectedArticle: SelectedArticle) {
         let updatedArticle = SelectedArticle(article: selectedArticle.article, unitIndex: idx, quantity: selectedArticle.quantity)
+        let quantity = min(selectedArticle.quantity, CheckoutSummaryArticleRefineType.quantity.count(selectedArticle: updatedArticle))
+        let fixedArticle = SelectedArticle(article: selectedArticle.article, unitIndex: idx, quantity: quantity)
```
Comment (bonkey) [inline AtlasUI/AtlasUI/CheckoutSummaryEditProductStackView.swift]: Oh, did `SelectedArticle` break or needs to stand still here, that we need to fix it? ;-)

It's only guilty of being **selected**.

Only quantity needs to be adjusted, and `quantity` name rather does not need to mention it's adjusted.

Comment (hani-ibrahim) [inline AtlasUI/AtlasUI/CheckoutSummaryEditProductStackView.swift]: now better? 😅

**Comment by hani-ibrahim** [comment]
👍

**Comment by bonkey** [comment]
👍

**Comment by hani-ibrahim** [comment]
👍

**Comment by ahmed-shehata** [comment]
👍

**Pull request merged**